<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>项目</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QTableView" name="taskList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_addTask">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>主题</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="selectLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>主题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_themelist"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>移除</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QComboBox" name="cmb_style">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>87</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Round</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Water</string>
        </property>
       </item>
      </widget>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>441</width>
         <height>351</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_round">
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>211</width>
           <height>231</height>
          </rect>
         </property>
         <property name="title">
          <string>编辑</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>线条颜色</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>线条宽度</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_linewidth">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>150</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>边框颜色</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>边框宽度</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_borderwidth">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>190</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>背景颜色</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_linecolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>27</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_bordercolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_bgcolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>110</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_water">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>211</width>
           <height>281</height>
          </rect>
         </property>
         <property name="title">
          <string>编辑</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>波浪密度系数</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>160</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>使用颜色</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_waterDensity">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>41</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>波浪高度系数</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>200</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>未使用颜色</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_waterHeight">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>70</y>
            <width>41</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>边框宽度</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_usedcolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>150</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_unUsedcolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>190</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_textcolor">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>230</y>
            <width>28</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>240</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>文本颜色</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="le_waterborderWidth">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>110</y>
            <width>41</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
